clover-core/src/test/resources/javasyntax1.15/java15/Java15TextBlock.java

```diff
@@ -0,0 +1,37 @@
+package java15;
+
+public class Java15TextBlock {
+    public void textBlock() {
+
+        System.out.println("""
+                """);
+
+        System.out.println("""
+                "" " " ""
+                """);
+
+        System.out.println("""
+                \" \"" \u005c" \u005c""
+                """);
+
+        System.out.println("""
+                \t \r \n \r \t
+                """);
+
+        System.out.println("""
+                \u005c\ \\u005c
+                """);
+
+        System.out.println("""
+                text \
+                concatinated""");
```

Comment:
```suggestion
                concatenated""");
```

Reply:
Thank you for pointing this out. I have fixed it.